src/app/utils/Log.ts

```diff
@@ -1,150 +1,106 @@
-import logdna, { Logger, LogOptions } from '@logdna/logger';
+import { pino } from 'pino';
 import apiKeys from '../service/constants/apiKeys';
 import { CommandContext } from 'slash-create';
 import * as Sentry from '@sentry/node';
 
-let logger: Logger;
+const transport = pino.transport({
+	target: 'pino-loki',
+	options: {
+		batching: true,
+		interval: 5,
+		basicAuth: {
+			username: apiKeys.lokiUserName,
+			password: apiKeys.lokiPassword,
+		},
+	},
+});
 
-try {
-	logger = logdna.createLogger(apiKeys.logDNAToken, {
-		app: apiKeys.logDNAAppName,
-		level: apiKeys.logDNADefault,
-	});
-	if (process.env.NODE_ENV != 'production' || !logger.info) {
```

Comment:
let's adapt this `if ... then` function/logging coverage to `pino` to capture setup errors.